SU-HSDO/suhumsci
# SHS-5812: Remove Webform module from the codebase

# Summary
Remove `hs_webform` and webform contrib module from codebase

## Need Review By (Date)
01/29

## Urgency
medium

## Steps to Test
1. Log into a [tugboat test site](https://pr1648-frfiilaa2mg89bd5rksk4uqrtfejhias.tugboatqa.com/user/login). Visit the [module list admin page](https://pr1648-frfiilaa2mg89bd5rksk4uqrtfejhias.tugboatqa.com/admin/modules) (_Manage > Extend_ in the toolbar)
2. Search for "webform" and confirm that the custom "H&S Webform Paragraph", the webform contrib module and all its submodule are not present in the module list (**Note:** you'll see 2 disabled modules: "Webform Event Dispatcher" and "Token Or Webform". Those are webform integration modules provided by another contrib module and can be ignored).

# Review Tasks

## Backend / Functional Validation
### Code

### Code security

### General
